uxarray/grid/coordinates.py

```diff
@@ -286,3 +286,50 @@ def _populate_lonlat_coord(grid):
             "long_name": "latitude of mesh nodes",
             "units": "degrees_north",
         })
+
+
+def grid_center_lat_lon(ds):
```

Comment:
I didn't understand why this is a user API if it is a SCRIP-specific function. Moreover, it seems like this was already here in the main branch, I don't know why the PR shows this as an added function. Any ideas?

Reply:
I'll fix this, it must of gotten moved around.